x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/errors/generate_data.ts

```diff
@@ -39,3 +39,38 @@ export function generateData({ from, to }: { from: number; to: number }) {
         .success(),
     ]);
 }
+
+export function generateMultipleErrorsData({
+  from,
+  to,
+}: {
+  from: number;
+  to: number;
+}) {
+  const range = timerange(from, to);
```

Comment:
Passing in the number of errors that you want to generate might make things a little more clear (instead of having "multiple" in the name):

```suggestion
export function generateErrors({
  from,
  to,
  errorCount = 50
}: {
  from: number;
  to: number;
  errorCount: number
}) {
  const range = timerange(from, to);
```